chart/values.yaml

```
@@ -176,6 +176,13 @@ resources: {}
   #  memory: 128Mi
   #
 
+metrics:
+  ## Enable the Prometheus ServiceMonitor.
+  enabled: false
```

The metrics are always exposed by default. Can we make it clear that this field enables the Prometheus service monitor?
```suggestion
  prometheusServiceMonitor:
    enabled: false
```